fury/ui/elements.py

```diff
@@ -3578,14 +3578,15 @@ class PlaybackPanel(UI):
        such as play, pause, stop, and seek.
     """
 
-    def __init__(self, loop=False, position=(0, 0)):
-        super(PlaybackPanel, self).__init__()
-        self.position = position
+    def __init__(self, loop=False, position=(0, 0), width=None):
```

Comment:
I wonder if make sense to have position at `(0,0)` instead of `None`

Reply:
I think it should be `(0, 0)` to keep consistency with other UI elements that has `position=(0, 0)`. That is true if the position is measured from the bottom left of the panel

Reply:
ok, fair enough. LGTM, merging